README.md

```diff
@@ -104,25 +198,67 @@ Otherwise, if your application is bundled via bundlers like `webpack`, you can i
 
 In case you just want to check out the latest `bundle.js` without installing the package, we publish one on each GitHub release. You can find it under [this link to the latest release](https://github.com/asyncapi/parser-js/releases/latest/download/bundle.js).
 
-## Custom message parsers
+## Custom schema parsers
+
+AsyncAPI doesn't enforce one schema format. We can have payload of your messages described with OpenAPI (3.0.0), Avro, etc. This parser by default parses only [AsyncAPI Schema Format](https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md#schemaObject) (superset of JSON Schema Format). We can extend it by creating a custom parser and registering it within the parser:
+
+1. Create custom parser module that exports three functions:
+
+    - `validate` - function that validates (its syntax) used schema.
+    - `parse` - function that parses the given schema to the [AsyncAPI Schema Format](https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md#schemaObject).
+    - `getMimeTypes` - function that returns the list of mime types that will be used as the `schemaFormat` property to determine the mime type of a given schema.
+
+    Example:
+    
+    ```ts
+    export default {
+      validate(input) { ... },
+      parse(input) { ... },
+      getMimeTypes() {
+        return [
+          'application/vnd.custom.type;version=1.0.0',
+          'application/vnd.custom.type+json;version=1.0.0',
+        ]
+      }
+    }
+    ```
 
-AsyncAPI doesn't enforce one schema format for messages. You can have payload of your messages described with OpenAPI, Avro, etc. This parser by default parses only AsyncAPI schema format. You can extend it by creating a custom parser and registering it within the parser:
+2. Before parsing/validating an AsyncAPI document with a parser, register the additional custom schema parser:
 
-1. Create custom parser module that exports two functions:
+    ```ts
+    import { Parser } from '@asyncapi/parser';
+    import myCustomSchemaParser from './my-custom-schema-parser';
 
-    ```js
-    // TBD
+    const parser = new Parser();
+    parser.registerSchemaParser(myCustomSchemaParser);
     ```
 
-2. Before parsing an AsyncAPI document with a parser, register the additional custom schema parser:
+### Official supported custom schema parsers
```

Comment:
I suggest we move this section to be at the top of `Custom schema parsers` section, as it will be easier for the user to find out info about those common parsers.

Reply:
After all, the introduction to this section is short, why move it up by these few lines?

Reply:
Because is what users will look for **always** when looking for schema parsers: "Is there already an official parser for <whatever>?"

Reply:
What can I say? if they are impatient then let them cancel their Netflix account 😆